core/domain/user_services.py

```diff
@@ -1062,6 +1240,39 @@ def set_username(user_id, new_username):
     _save_user_settings(user_settings)
 
 
+def set_user_display_alias(user_id, new_display_alias):
```

Comment:
Can you explain the use case for this?

Reply:
This will used in the controller to set/update the display alias. Similar to the current `PreferencesHandler` controller ([here](https://github.com/oppia/oppia/blob/develop/core/controllers/profile.py#L176-L205)), we can make another controller who updates all the applicable attributes as received in the request.
This will help us to use the existing functions in user_services to update the user data.

Reply:
Is one single controller going to update all of these individually? As discussed yesterday, that would cause us to update the model multiple times right?

Reply:
Yes that would be unwanted because for profiles we want to updates all attributes in one go.

So I have made a new function now which has all the user data for a single user (full/profile) - and that updates all the attributes. We will use this function for android only.